Jenkinsfile

```diff
@@ -168,25 +190,31 @@ podTemplate(name: 'fedora-atomic-' + env.ghprbActualCommit,
                         // Send message org.centos.prod.ci.pipeline.compose.running on fedmsg
                         pipelineUtils.sendMessage(messageFields['properties'], messageFields['content'])
 
-                        // Provision resources
-                        pipelineUtils.provisionResources(currentStage)
+                        // Execute in containers
+                        env.rsync_paths = "ostree"
 
-                        // Stage resources - ostree compose
-                        pipelineUtils.setupStage(currentStage, 'fedora-atomic-key')
+                        dir('ci-pipeline') {
+                            env.rsync_from = "${env.RSYNC_USER}@${env.RSYNC_SERVER}::${env.RSYNC_DIR}/${env.branch}/"
+                            env.rsync_to = "/home/output/"
 
-                        // Rsync Data
-                        pipelineUtils.rsyncData(currentStage)
+                            pipelineUtils.executeInContainer(currentStage + "-rsync-before", "rsync", "/tmp/rsync.sh")
+
+                            pipelineUtils.executeInContainer(currentStage, "ostree-compose", "/tmp/ostree-compose.sh")
+
+                            env.rsync_to = "${env.RSYNC_USER}@${env.RSYNC_SERVER}::${env.RSYNC_DIR}/${env.branch}/"
+                            env.rsync_from = "/home/output/"
 
-                        def ostree_props = "${env.ORIGIN_WORKSPACE}/logs/ostree.props"
-                        def ostree_props_groovy = "${env.ORIGIN_WORKSPACE}/ostree.props.groovy"
+                            pipelineUtils.executeInContainer(currentStage + "-rsync-after", "rsync", "/tmp/rsync.sh")
+                        }
+
+
+                        def ostree_props = "${env.WORKSPACE}/ci-pipeline/" + currentStage + "/logs/ostree.props"
+                        def ostree_props_groovy = "${env.WORKSPACE}/ostree.props.groovy"
                         pipelineUtils.convertProps(ostree_props, ostree_props_groovy)
                         load(ostree_props_groovy)
 
-                        // Teardown resource
-                        pipelineUtils.teardownResources(currentStage)
-
                         // Set our message topic, properties, and content
-                        messageFields = pipelineUtils.setMessageFields("package.complete")
+                        messageFields = pipelineUtils.setMessageFields("compose.complete")
```

Comment:
Good catch on this, we should have a compose.queued message in the previous step too. Nit pick on the comment in the line below this as well.